Documentation/help_files/mfa_help.php

```diff
@@ -0,0 +1,319 @@
+                            
+                            <p><?php echo xlt("The application generates challenge - a random number and passes it to the USB key along with the nonce and checksum stored in the application");?>.
+                            
+                            <p><?php echo xlt("The USB key takes this the nonce supplied by the application after confirming the checksum and generates a private key using the process similar to that used in the registration process");?>.
+                            
+                            <p><?php echo xlt("It uses this private key to sign the challenge and sends the response back to the application");?>.
+                        
+                            <p><?php echo xlt("The application then uses the public key that the USB device sent on registration to verify the response");?>.
+                            
+                            <p><?php echo xlt("If successful it authenticates the user");?>.
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div class= "row" id="section2">
+                <div class="col-sm-12">
+                    <h4 class="oe-help-heading"><?php echo xlt("Advantages and Disadvantages of TOTP and U2F"); ?><a href="#top_section"><i class="fa fa-arrow-circle-up oe-pull-away oe-help-redirect" aria-hidden="true"></i></a></h4>
+                    <p><?php echo xlt("TOTP - Pros"); ?>:
+                        <ul>
+                            <li><?php echo xlt("Free authenticator app"); ?></li>
+                            <li><?php echo xlt("Easy to setup"); ?></li>
+                        </ul>
+                        
+                    <p><?php echo xlt("TOTP - Cons"); ?>:
+                        <ul>
+                            <li><?php echo xlt("Needs a Smartphone"); ?></li>
+                            <li><?php echo xlt("If phone is lost will not be able to login"); ?></li>
+                            <li><?php echo xlt("Being a software application it can be compromised by malware that steals the secret key or by social engineering"); ?></li>
+                        </ul>
+                        
+                    <p><?php echo xlt("U2F - Pros"); ?>:
+                        <ul>
+                            <li><?php echo xlt("Easy to use"); ?></li>
+                            <li><?php echo xlt("Very secure"); ?></li>
+                        </ul>
+                        
+                    <p><?php echo xlt("U2F - Cons"); ?>:
+                        <ul>
+                            <li><?php echo xlt("Many companies block operations with USB ports on corporate computers"); ?></li>
+                            <li><?php echo xlt("U2F devices are relatively costly"); ?></li>
```

Comment:
Can get U2F for $6 : https://www.google.com/search?q=amazon+u2f&safe=off&sa=X&biw=1869&bih=990&tbm=shop&tbs=p_ord:p&ei=mSi8XI27LqbZ0gLXuZPIDQ&ved=0ahUKEwjN65CU4eDhAhWmrFQKHdfcBNkQuw0IogIoAg

Reply:
I will remove $ amount. $6 internationally might still be too high, authenticator apps are free :)

Reply:
Done